src/lib/math.ts

```diff
@@ -8,7 +8,7 @@ import { bn, bnOrZero } from './bignumber/bignumber'
 // - If displayDecimals are provided, it will use that precision, to return e.g a human, precision, or number stripped to any arbitrary decimal places
 // and use the ROUND_DOWN rounding mode
 export const fromBaseUnit = (
-  value: BigNumber.Value,
+  value: BigNumber.Value | undefined,
```

Comment:
remove the need for optional chaining to show `0` value. Note `bnOrZero` on value to make this safe.